### ISSUE TITLE
Any ImageNet pretrained weights?

### ISSUE THREAD

**Issue by CauchyComplete** [sithu31296/semantic-segmentation#34]
Thanks for this awesome repo!
Are there any ImageNet pretrained weights?
Thanks

**Comment by sithu31296** [comment]
Take a look at [this](https://github.com/sithu31296/semantic-segmentation/blob/main/docs/BACKBONES.md). All the weights provided there are trained on imagenet.

**Comment by CauchyComplete** [comment]
Thanks. Then no ImageNet weights for BiSeNet v2?
Plus, could you tell me how to combine encoders with heads?

**Comment by sithu31296** [comment]
Yes. BiSeNetv2 doesn't use a separate backbone model. So, no ImageNet weights for it. 
For custom model creation, please see the sample from [here](https://github.com/sithu31296/semantic-segmentation/blob/main/semseg/models/custom_cnn.py). Basically, you need to know the output channels of an encoder since a head class needs at least 3 parameters (channel dims, head dim, num_classes).

**Comment by scl666** [comment]
> 是的。BiSeNetv2 不使用单独的主干模型。因此，没有图像网络的权重。有关自定义模型的创建，请参阅[此处](https://github.com/sithu31296/semantic-segmentation/blob/main/semseg/models/custom_cnn.py)的示例。基本上，您需要知道编码器的输出通道，因为 head 类至少需要 3 个参数（通道暗淡、磁头调暗num_classes）。

你好，camvid数据集使用时损失出现nan，cityscapes没有，博主知道啥情况吗

**Comment by zsSWJTU** [comment]
> Take a look at [this](https://github.com/sithu31296/semantic-segmentation/blob/main/docs/BACKBONES.md). All the weights provided there are trained on imagenet.

Hello, thank you for contributing your code. However, the link to the pre-trained weights you provided appears to be broken. How can I download these weights?

**Comment by FreemanTang** [comment]
> > Take a look at [this](https://github.com/sithu31296/semantic-segmentation/blob/main/docs/BACKBONES.md). All the weights provided there are trained on imagenet.
> 
> Hello, thank you for contributing your code. However, the link to the pre-trained weights you provided appears to be broken. How can I download these weights?

@sithu31296 Hello, boss, I have the same problem. Your download link for pre-training weights is invalid. Is there a new download link?
